httpgrpc/README.md

```diff
@@ -0,0 +1,5 @@
+To rebuild generated protobuf code, run:
+
+    protoc -I ./ --go_out=plugins=grpc:./ ./httpgrpc.proto
```

Comment:
This repo is vendored in, and the vendoree (is that a word) probably won't build the proto.  So I checked in the build code and put instructions for updating it here.